docs/how-to-guides/advanced-user-guides/enable-experimental-features/sqlite-caching.md

```diff
@@ -0,0 +1,41 @@
+---
+title: SQLite-Backed Caching
```

Comment:
```suggestion
title: SQLite-Backed Caching and UI Server-Side Pagination
```

Wonder if we need to make this a little clearer i.e backend cache changes and frontend changes that make use of it. I've tweaked a few things below, but could we call this out in the title?

Reply:
From a user perspective I actually think only UI is relevant - use of SQLite and fact that pagination is computed "Server-side" are implementation details.

From an engineering perspective, we are changing the cache implementation to enable better pagination. Both aspects are equally important and have to go hand in hand. I do not care which one is given more prominence.

How about just using

```suggestion
title: UI Server-Side Pagination
```

And clarifying later that is achieved via a new SQLite cache?

Reply:
Sounds good!